sklearn/ensemble/tests/test_bagging.py

```diff
@@ -663,3 +663,21 @@ def test_oob_score_removed_on_warm_start():
     clf.fit(X, y)
 
     assert_raises(AttributeError, getattr, clf, "oob_score_")
+
+
+def test_consistent_index_sampling():
```

Comment:
maybe also check that when fixing the random state, the results are identical? (we should really have that as a common test though)